exec-healthz/build/build.sh

```diff
@@ -16,6 +16,11 @@
 
 set -o errexit
 set -o nounset
+# For ubuntu updating bash to avoid illegal instruction error for pipefail
```

Comment:
Is this really required here or was it just that you had an old version of bash yourself?

Can we just remove pipefail? Is it really required?